src/Cassandra.IntegrationTests/TestClusterManagement/SimulacronManager.cs

```diff
@@ -104,7 +104,7 @@ public void Stop()
 
         public bool IsUp()
         {
-            return _started;
+            return _started && _simulacronProcess != null && !_simulacronProcess.HasExited;
```

Comment:
I think using the `_started` flag as it was, it was easier to understand (started or manually stopped and can be restarted).

Maybe it would be best to remove this method (not used outside this class) and add it if needed in the future when we define a purpose for it.